src/nvc.rs

```diff
@@ -479,8 +609,10 @@ impl Nvc {
                         OPCODE_SYSTEM_REGISTER_ID_ECR => self.reg_ecr as _,
                         OPCODE_SYSTEM_REGISTER_ID_PSW => self.reg_psw(),
                         OPCODE_SYSTEM_REGISTER_ID_CHCW => {
-                            logln!("WARNING: stsr chcw not yet implemented");
```

Comment:
Same as above; I think we should leave this msg in here as a note-to-self (as "not fully implemented").